Add VDS13/telegram-inline-calendar (#37)
grammyjs/awesome-grammY: README.md

- [Web Apps](#web-apps)
- [Templates](#templates)
- [Learning Resources](#learning-resources)

> - [Libraries](#libraries)

Reviewer: ```suggestion
- [Plugins](#plugins)
```

## Learning Resources

- [Desarrollo de bots de Telegram en TypeScript](https://www.youtube.com/playlist?list=PL7q-McYJyHlip2-k4RW1nDxJWsFuO4nO1) - A free course in Spanish on creating bots using grammY.

> ## Libraries

Reviewer: ```suggestion
## Plugins
```

> - [VDS13/telegram-inline-calendar](https://github.com/VDS13/telegram-inline-calendar) - Date and time picker and Inline calendar for Node.js telegram bots.

Reviewer: ```suggestion
- [VDS13/telegram-inline-calendar](https://github.com/VDS13/telegram-inline-calendar) - Date picker and inline calendar for your bots.
```
